index.html

```diff
@@ -14,31 +14,23 @@
   </head>
 
   <body>
+    <section>
     <!--header-->
     <header class="header">
-<<<<<<< HEAD
-      <img class="logo" src="./logo.svg" alt="LOGO" />
+      
+      <div class="logo">
+        <a href="index.html"><img  src="./logo.svg" alt="LOGO" /></a>
+      </div>
 
       <nav>
         <a class="dropdown" href="">
           <i class="fa fa-bars" aria-hidden="true"> </i
         ></a>
       </nav>
-=======
-        
-        <img class="logo" src="./logo.svg" alt="MY LOGO">
-                          
-         <nav >
-             <a class="dropdown" href="">
-                 <i class="fa fa-bars" aria-hidden="true">
-                 </i></a>
-        </nav>
-        
->>>>>>> 58d53c1c6a143b75c4a37ad42bf8a2ced709bf0a
-    </header>
+   </header>
 
     <!--Introdution section-->
-    <main class="main">
+    <div class="main">
```

Comment:
- [ ] As the previous reviewer mentioned Kindly wrap the whole headline section with a `<section>` tag instead of using `<div>` tag for semantic reasons.